pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -237,6 +237,8 @@ metadata:
   name: jiva-volume-delete-default
 spec:
   defaultConfig:
+  - name: ScrubImage
+    value: {{env "OPENEBS_IO_SCRUB_IMAGE" | default "quay.io/openebs/openebs-tools:latest"}}
```

Comment:
Why is the version changed?

Reply:
if the any image is not found in env variable we should go with the latest version.

Reply:
Can you make the version as 3.8? (There is no latest tag for this openebs-tools, which is a busybox image - retagged. )

Reply:
done